front/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "scripts": {
-    "dev": "next dev",
+    "dev": "concurrently --kill-others \"cd ../types/ && npm run start\" \"sleep 10s && cd ../sdks/js/ && npm run start\" \"sleep 12s && next dev\"",
```

Comment:
This will watch all the dependencies included via "file://..." in parallel so we won't have to do it ourselves.

Reply:
I'm not really fond of this, restarting front will restart recompile types and sdks everytime and we'll have to wait 12s before the app starts ? I restart front dizains of times per day and don't really want to recompile types and wait everytime. Although I agree we could have one script to start the compilation of types and sdks at the same time., but that should be separate from the `next dev`
Btw `sleep 10s` does not work for me, it has to be `sleep 10` ?

Reply:
> I'm not really fond of this, restarting front will restart recompile types and sdks everytime and we'll have to wait 12s before the app starts ? I restart front dizains of times per day and don't really want to recompile types and wait everytime. Although I agree we could have one script to start the compilation of types and sdks at the same time., but that should be separate from the next dev

I never restart front on my side but fair :)
We can have 3 scripts:
- one to do it all at once (my use case)
- one to simply `next dev` (current behavior, default)
- one to start types & sdk in //.

Will update.

> Btw sleep 10s does not work for me, it has to be sleep 10 ?

```
➜  ~ sleep
usage: sleep number[unit] [...]
Unit can be 's' (seconds, the default), m (minutes), h (hours), or d (days).
```

Are you using zsh (i am) ?

Reply:
yes, zsh, but seems a command for me :
```
➜  ~ sleep
usage: sleep seconds
➜  ~ which sleep
/bin/sleep
```

:shrug: